Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.13 as builder
+FROM golang:1.14 as builder
```

Comment:
Q: did you consider 1.15 ? https://github.com/AbsaOSS/samlet/blob/main/go.mod#L3